dts/bindings/sensor/invensense,icm42670-i2c.yaml

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2022 Esco Medical ApS
+# Copyright (c) 2020 TDK Invensense
+# SPDX-License-Identifier: Apache-2.0
+
+description: ICM-42670 motion tracking device
+
+compatible: "invensense,icm42670"
+
+include: [sensor-device.yaml, i2c-device.yaml]
```

Comment:
A common yaml file e.g. `icm42670-common.yaml` should be created with all the common properties and then included here.